photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/GaussianProcessEstimator.scala

```diff
@@ -38,11 +36,14 @@ import com.linkedin.photon.ml.hyperparameter.estimators.kernels._
 class GaussianProcessEstimator(
     kernel: Kernel = new RBF,
     normalizeLabels: Boolean = false,
+    noisyTarget: Boolean = false,
     predictionTransformation: Option[PredictionTransformation] = None,
     monteCarloNumBurnInSamples: Int = 100,
-    monteCarloNumSamples: Int = 100,
+    monteCarloNumSamples: Int = 10,
```

Comment:
I reduced this because I noticed that the default in spearmint is 10, and I ran a few small experiments and it didn't seem to make much difference. So it makes sense if it reduces computational burden. If you feel strongly about it, I'll change it back.

Reply:
I think we can keep it and test things out. And maybe change later if need be.